### DISCUSSION TITLE
RGB Out

### DISCUSSION THREAD

**Discussion by petabyteHD** [UzixLS/zx-sizif-512#12] [Q&A]
Hi, where can i find a rgb pinout to make a scart cable?

**Comment by UzixLS** [comment]
RGB pinout is same as on Sega Mega Drive 2.
![изображение](https://user-images.githubusercontent.com/1556813/167424234-65961db4-2ffa-4b40-a1f5-ea21f3e1a9d5.png)

**Reply by petabyteHD** [reply]
thanks!!!

**Reply by oO-cozy-Oo** [reply]
Would a ZX Spectrum +2 Grey ISS3, RGB to VGA Adaptor for 15kHz monitor screen display work on the sizif
![s-l1600](https://user-images.githubusercontent.com/112904233/188510225-3da91879-a48d-4475-b781-4a3b76966a6f.jpg)

**Reply by UzixLS** [reply]
No, it uses a different connector.

**Reply by oO-cozy-Oo** [reply]
Found this maybe this will work for a vga out
https://github.com/jeffqchen/9DIN2VGA

**Comment by oO-cozy-Oo** [comment]
![01](https://user-images.githubusercontent.com/112904233/190926384-6fb84c85-32bd-4875-9cf5-db5bee60e5dc.jpg)
